cmd/generate.go

```diff
@@ -132,7 +147,7 @@ func (gc *generateCmd) loadAPIModel() error {
 			Locale: gc.locale,
 		},
 	}
-	gc.containerService, gc.apiVersion, err = apiloader.LoadContainerServiceFromFile(gc.apimodelPath, true, false, nil)
```

Comment:
The existing implementation runs a full ContainerService validation (above). The new one seems only to run a vlabs ContainerService validation. Is that intentional?

Reply:
this change is equivalent to one we made for deploy https://github.com/Azure/aks-engine/pull/1429/files

Reply:
that is correct, I followed the pattern in deploy command.